scene/main/node.h

```diff
@@ -597,6 +605,25 @@ class Node : public Object {
 
 	virtual String to_string() override;
 
+	/* USED ONLY BY PACKED SCENE */
+
+	void add_connection_owner(Node *owner, Node *to_node, const StringName &signal_name, Callable c) {
+		ConnectionOwnerData sc;
+		sc.to_node = to_node;
+		sc.signal_name = signal_name;
+		sc.method_name = c.get_method();
+		data.connection_owners[owner] = sc;
+	}
+
+	Node *get_connection_owner(Node *to_node, const StringName &signal_name, Callable c) const {
```

Comment:
```suggestion
	Node *get_connection_owner(Node *p_to_node, const StringName &p_signal_name, const Callable &p_callable) const {
```
Also why is the definition here, not in cpp?

Reply:
No reason. I'll move it.